tests/test_correct_entries.py

```diff
@@ -0,0 +1,13 @@
+import argparse
+from pathlib import Path
+
+from validator.checker import UpdateChecker
+from validator import get_latest_version
+
+def test_valid_sql_sentences():
+    root_dir = Path(__file__).parents[1]
+    upgrade = root_dir / 'updates' / str(get_latest_version()) / 'updates.sql'
```

Comment:
This will only test the last version file. The test should run on **all** updates. I think you need to read all ofthem and pytest.parametrize the test.